server/src/concepts/user/controllers/index.js

```diff
@@ -49,3 +44,18 @@ export const deleteUser = async (req, res) => {
   }
   return res.status(200).send(`User with ${req.params.id} was deleted`);
 };
+
+export const loginUser = async (req, res) => {
+  try {
+    const token = await signInUserFunc(req.body.email, req.body.password);
+    if (!token) {
+      return res.status(400).send("Unsuccessful login attempt");
+    }
+    res.setHeader("authorisation", token);
```

Comment:
```suggestion
    res.setHeader("token", token);
```

token instead of authorisation because when we are expecting "token" header in request, i think it is better to keep it consistent